sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Bug Fixes
 - Ensured that `RetryPolicy` and `HttpLogOptions` use a default implementation when creating Key Vault clients if not set or set to `null`.
 
+### Breaking Changes
+- Removed service method overloads that take a `pollingDuration`, since `PollerFlux` and `SyncPoller` objects allow for setting this value directly on them.
```

Comment:
```suggestion
- Removed service method overloads that take a `pollingInterval`, since `PollerFlux` and `SyncPoller` objects allow for setting this value directly on them.
```